### DISCUSSION TITLE
Show note field in bibliography

### DISCUSSION THREAD

**Discussion by Cano707** [typst/typst#2377] [Q&A]
Hi everyone,

I am looking for a way to display the note field defined in the entries of the bib file in the bibliography.
Say I have the following bib entry:
```
@misc{ASYNC,
 author = {{SomeAuthor}},
 note = {Written by Someone},
 year = {2023},
 url = {https://some.url},
 urldate = {2023-09-26}
}
```

How do I make the note appear in the resulting file?

**Comment by gdietl** [comment]
Did you come up with a solution? I have exactly the same problem. Notes do not show up, nor does the institution in Technical Reports.

**Reply by Cano707** [reply]
Unfortunately not, I would be crucial to implement this feature, but I can't wait for it, since I have due dates.

However, if you find a solution, please let me know.

**Reply by gdietl** [reply]
I found the problem and a workaround. When converting the bib-files to yml-files using hayagriva, the notes are lost. Adding the notes, e.g., as the entry type "publisher" (unfortunately, the entry type notes does not exist) makes them visible.

**Reply by Cano707** [reply]
I see. Thank you for sharing!

**Comment by yochem** [comment]
You could file an issue at [typst/hayagriva](/typst/hayagriva).

**Reply by Cano707** [reply]
You are right. I will probably do so later.

**Comment by Saturnine-Softworks** [comment]
Yeah I would note that Hayagriva *does* accommodate a notes field – it seems that Typst just never renders it. And I'm writing directly in hayagriva format in a yaml file rather than in BibTeX. 

Edit: This is extremely unideal but for now I'm getting around this with
```yaml
serial-number: "\n<note>"
```
because like @Cano707, I have due dates. 

Edit 2: This doesn't work for all citation types, because some place the publisher + date last, and you can't just sub in an arbitrary string for the date. Unfortunately I think this means the only "workaround" is to just not use the `#bibliography()` function.
